### PR TITLE
refactor(experimental): graphql: refactor transforms out of loaders and into resolvers

### PR DESCRIPTION
**Pull request by buffalojoec**
Simply refactoring the transformers out of the loaders and into the resolvers.

The loaders should be responsible for loading RPC responses, that's it.

Any necessary transformations can happen at the resolver level.

### CODE REVIEW

**Comment by buffalojoec** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2064?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `master`
  * **PR #2045** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2045?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #2046** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2046?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #2064** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2064?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
        * **PR #2047** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2047?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #2063** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2063?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #2048** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2048?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2064?utm_source=stack-comment).

File: packages/rpc-graphql/src/loaders/block.ts [hunk 1]
```diff
@@ -1,72 +1,48 @@
...
-/* Load a block from the RPC, transform it, then return it */
-async function loadBlock(rpc: Rpc, { slot, ...config }: ReturnType<typeof normalizeArgs>) {
-    const block = await rpc
+async function loadBlock(rpc: Rpc<GetBlockApi>, { slot, ...config }: BlockLoaderArgs): Promise<BlockLoaderValue> {
+    // @ts-expect-error // FIX ME: https://github.com/solana-labs/solana-web3.js/pull/2052
+    return await rpc
         .getBlock(
             slot,
-            // @ts-expect-error FIXME: https://github.com/solana-labs/solana-web3.js/issues/1984
+            // @ts-expect-error // FIX ME: https://github.com/solana-labs/solana-web3.js/pull/2052
```
Comment (steveluscher) [inline packages/rpc-graphql/src/loaders/block.ts:30]: W HY?

**Review by steveluscher — APPROVED** [review]
Added via Giphy

File: packages/rpc-graphql/src/loaders/block.ts (hunk 1 above)
Comment (buffalojoec) [reply packages/rpc-graphql/src/loaders/block.ts:30]: Oopsie lol.

**Comment by buffalojoec** [comment]
### Merge activity

* **Jan 27, 5:31 PM**: @buffalojoec started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2064).

**Pull request merged**